src/Tribe/Customizer.php

```diff
@@ -583,11 +583,13 @@ public function register( WP_Customize_Manager $customizer ) {
 			 * Allows people to Register and de-register the method to register more Fields
 			 *
 			 * @since 4.4
+			 * @since TBD Add Customizer instance as a parameter.
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: e349c0ca78e93bbf0b5c7dfabee81e0b</sub>